lib/node/patch.js

```diff
@@ -100,6 +101,12 @@ export default function patchNode(element, newHTML, options) {
     };
   }
 
+  if (newTree && newTree.childNodes) {
+    newTree.childNodes = normalizeChildNodes(newTree.childNodes);
```

Comment:
This call is very very expensive and I'll be refactoring it out. Incurs a 5ms penalty during benchmarking.